tests/e2e-openshift/red-metrics/chainsaw-step-03-assert-1-1.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: apps/v1
```

Comment:
Why is this assert (03-assert.yaml) split into 6 different files?

Reply:
The migration was done with chainsaw v0.1.3 that had this behaviour, it was fixed in v0.1.4.

For now i will merge them manually.